### ISSUE TITLE
Lacking GNOME Shell and MATE support

### ISSUE THREAD

**Issue by Baemir** [snwh/humanitary-gtk-theme#6]
I love this and I think it would be awesome if you made a Shell theme and a Metacity theme for it so that we could use it outside of eOS.

**Comment by snwh** [comment]
This was more of a one-off theme than something I want to maintain extensively and starting down that route would be a lot of extra work.

**Comment by Baemir** [comment]
Oh well. In that case I may try to come up with something when I feel like spending time on theming, or just install eOS lol. That said I sincerely believe this theme could catch on if you put in the work, because nowadays there are virtually zero up to date themes for GNOME other than horst's, and I think a lot of people missed these riskier color schemes. But I respect your choice :).

**Comment by snwh** [comment]
I have a couple other GTK and icon themes that I maintain, is all. ;)

**Comment by Milozzy** [comment]
@Baemir please consider supporting Ubuntu LTS too if you'll fork this wonderful theme... A human like theme would be a perfect match for Ubuntu. Thanks

**Comment by snwh** [comment]
This project isn't being updated, it was a one-time thing.

**Comment by Milozzy** [comment]
Yeah I saw it unfortunately. Don't wanted to bother you, just want to let Baemir know that if he wil "come up with something" it would be much appreciated :)
